src/status_im/ui/screens/wallet/choose_recipient/views.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im.ui.screens.wallet.choose-recipient.views
```

Comment:
this view doesn't exist anymore in develop

Reply:
Makes sense now that I look at it again.  I'll try it again later today.